mora/service/orgunit.py

```diff
@@ -205,6 +205,16 @@ def get_one_orgunit(c, unitid, unit=None,
     elif details is UnitDetails.FULL:
         unittype = common.get_uuid(rels['enhedstype'][0], required=False)
 
+        if rels['overordnet'][0]['uuid']is not None:
+            parent = get_one_orgunit(c, rels['overordnet'][0]['uuid'],
+                                     details=UnitDetails.FULL)
+            if parent is not None:
+                path = parent['path'] + '/' + parent['name']
+            try:
```

Comment:
Generally speaking, `NameError`, like `SyntaxError`, shouldn't be raised as part of normal execution of the code. Given that you only assign to `path` in the prior conditional, please fold this `try`/`except` into the conditional prior to it.

Reply:
This I new to me. My understanding of Python is that exceptions are routinely used to mark the end of an iteration? In this case it indicates that the recursion has reached the top level. I can write it in another way if you want, but I think this is a quite common way to do it.

https://www.python.org/dev/peps/pep-0234/
https://jeffknupp.com/blog/2013/02/06/write-cleaner-python-use-exceptions/

Reply:
...ahhh now I see it...ok in this case I can do without the exception :-)
But I still do not agree that exceptions should not be part of normal execution.

Reply:
Well, I think you misunderstood me; I did not mean to say that _all_ exceptions should be avoided, but rather than specifically `NameError` and `SyntaxError` 🙂 

`StopIteration` is an interesting case, by the way, and has some rather odd side effects — at least without a `__future__` import, see [PEP-479](](https://www.python.org/dev/peps/pep-0479/). (We should consider adding that `__future__` import…)

Speaking of exceptions, this is a common antipattern:

```python
try:
  x = mapping[42]
except KeyError:
  x = 1337
```

Instead, do this:

```python
  x = mapping.get(42, 1337)
```

…but that doesn't imply that `KeyError` is evil or anything like that — it's very useful for non-trivially handling a missing value.